kof-operator/webapp/collector/src/components/pages/collectorPage/components/collector-list/CollectorsList.tsx

```diff
@@ -0,0 +1,49 @@
+import { useCollectorMetricsState } from "@/providers/collectors_metrics/CollectorsMetricsProvider";
+import { JSX } from "react";
+import CollectorsTable from "./CollectorsTable";
+import { Loader } from "lucide-react";
+import { Button } from "@/components/generated/ui/button";
+
+const CollectorsList = (): JSX.Element => {
+  const { data, isLoading, error, fetch } = useCollectorMetricsState();
+
+  if (isLoading) {
+    return (
+      <div className="flex w-full justify-center items-center mt-[15%]">
+        <Loader className="animate-spin w-8 h-8"></Loader>
+      </div>
+    );
+  }
+
+  if (!isLoading && !data) {
+    return (
+      <div className="flex w-full h-screen justify-center items-center">
+        You don't have any clusters
```

Comment:
Please use less directive phrases, just describing a state, instead of judging the user :)
```suggestion
        No clusters found
```